colcon_bundle/verb/assets/v2_setup.sh.jinja2

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env {{ shell }}
+
+# If using sh environment, please set BUNDLE_CURRENT_PREFIX before sourcing this script
+if [ "{{ shell }}" = "bash" ]; then
+	DIR="$( cd "$( dirname "$BASH_SOURCE" )" && pwd)"
+else
+	DIR="$BUNDLE_CURRENT_PREFIX"
+fi
+
+export PATH="$DIR/usr/sbin:$DIR/usr/bin:$DIR/usr/local/bin:$DIR/sbin:$DIR/bin:$PATH"
+export LD_LIBRARY_PATH="$DIR/lib:$DIR/usr/lib:$DIR/lib/x86_64-linux-gnu:$DIR/usr/lib/x86_64-linux-gnu:$DIR/usr/lib/x86_64-linux-gnu/mesa:$LD_LIBRARY_PATH:$DIR/lib/arm-linux-gnueabihf:$DIR/usr/lib/arm-linux-gnueabihf:$LD_LIBRARY_PATH:$DIR/lib/aarch64-linux-gnu:$DIR/usr/lib/aarch64-linux-gnu"
+
+export LC_ALL=C.UTF-8
+
+# source for ros 1
```

Comment:
I'd refactor this in a different PR

Reply:
Makes sense. Will add a follow up PR to improve the template, keeping it simple for now